cmd/podsync/config_test.go

```diff
@@ -80,6 +80,7 @@ timeout = 15
 	assert.EqualValues(t, "regex for title here", feed.Filters.Title)
 	assert.EqualValues(t, 0, feed.Filters.MinDuration)
 	assert.EqualValues(t, 86400, feed.Filters.MaxDuration)
+	assert.EqualValues(t, 265, feed.Filters.MaxAge)
```

Comment:
This tests is failing on CI

Reply:
Hi, thanks, I fixed both CI findings and force-pushed updated commit